internal/testrunner/reporters/xunit.go

```diff
@@ -0,0 +1,96 @@
+package reporters
+
+import (
+	"encoding/xml"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/testrunner"
+)
+
+func init() {
+	testrunner.RegisterReporter(XUnitReporter, ReportXUnit)
+}
+
+const (
+	// XUnitReporter reports test results in the xUnit format
+	XUnitReporter testrunner.TestReporter = "xUnit"
+)
+
+type testSuites struct {
+	XMLName xml.Name    `xml:"testsuites"`
+	Suites  []testSuite `xml:"testsuite"`
+}
+type testSuite struct {
+	Name    string      `xml:"Name,attr"`
+	Comment string      `xml:",comment"`
+	Suites  []testSuite `xml:"testsuite,omitempty"`
+	Cases   []testCase  `xml:"testcase,omitempty"`
+}
+type testCase struct {
+	Name string        `xml:"Name,attr"`
+	Time time.Duration `xml:"Time,attr"`
+
+	Error   string `xml:"Error,omitempty"`
+	Failure string `xml:"Failure,omitempty"`
+}
+
+// ReportXUnit returns the given test results formatted in the xUnit format
+func ReportXUnit(results []testrunner.TestResult) (string, error) {
```

Comment:
Same here (should it be exported?)

Reply:
No, it doesn't need to be. Good catch, will fix.

Reply:
Fixed in dbc6931bfd641a1e9fa45fd0c271afcda4109e9c.